bin/physlr-make

```diff
@@ -185,9 +186,11 @@ help:
 	@echo "	bloom_filter_size		size of bloom filter [10000000000] (10G)."
 	@echo ""
 	@echo "Example: To generate a physical map using physlr with stLFR reads myReads.fq.gz, run:"
-	@echo "	./physlr-make physical-map lr=myReads minimizer_overlap=stLFR"
+	@echo "	./physlr-make physical-map lr=myReads protocol=stLFR"
```

Comment:
Let's update these examples to reflect Shaun's naming suggestion while we're here to avoid using capitals. 
Ie. `myreads.fq.gz` instead of `myReads.fq.gz`